frontend/src/components/BreadcrumbNav.jsx

```diff
@@ -10,83 +10,115 @@ import {
 } from './BreadcrumbNav.styles';
 
 const BreadcrumbTeams = () => {
-    return <TeamsLink to={`/team`}>Teams</TeamsLink>;
+    return (
+        <li>
+            <TeamsLink to={`/team`}>Teams</TeamsLink>
+        </li>
+    );
 };
 
-const BreadcrumbItem = () => {
+const BreadcrumbItemTeam = props => {
     const { name } = useParams();
     const [{ selectedBranchState }] = useStateValue();
     const teamName = name || selectedBranchState.team;
-
     return (
-        <StyledInnerDiv>
-            <BreadcrumbTeams /> <span>/</span>
-            <StyledLink
-                to={`/team/${teamName}`}
-                className="TeamBreadCrumb"
-                id="TeamBreadCrumb"
-            >
-                {teamName}
-            </StyledLink>
-        </StyledInnerDiv>
+        <React.Fragment>
+            <BreadcrumbTeams />
+            <li>
+                <span>/</span>
```

Comment:
I think it would be wise to hide the "/"s from the screen reader.

Reply:
Lets' add this to TODO list